src/Sylius/Behat/NotificationType.php

```diff
@@ -59,6 +59,14 @@ public static function success()
     }
 
     /**
+     * @return NotificationType
+     */
+    public static function information()
+    {
+        return static::getTyped('information');
+    }
```

Comment:
I don't think we should use notification checker for empty cart notification. I'd be fine with just checking that there is text "Your cart is empty". I commented on the step so that should clarify.